src/config/config_def.c

```diff
@@ -406,6 +407,7 @@ static const WT_CONFIG_CHECK confchk_WT_SESSION_query_timestamp[] = {
 
 static const WT_CONFIG_CHECK confchk_WT_SESSION_reconfigure[] = {
   {"cache_cursors", "boolean", NULL, NULL, NULL, 0},
+  {"cache_max_wait_ms", "int", NULL, "min=0", NULL, 0},
```

Comment:
session_max_cache_wait_ms might be better a name ?

Reply:
Conventionally we don't append config with session or connection.